include/envoy/http/codec.h

```diff
@@ -359,8 +361,10 @@ class Connection {
   /**
    * Dispatch incoming connection data.
    * @param data supplies the data to dispatch. The codec will drain as many bytes as it processes.
+   * @return Envoy::Http::Status indicating the status of the codec. Holds any errors found in the
+   * callbacks.
```

Comment:
Maybe something like this: "Holds any errors encountered during processing of incoming data" ?

Reply:
done